lib/src/intl_suggestors/intl_message_syntax.dart

```diff
@@ -142,6 +142,22 @@ class MessageSyntax {
   String nameForNode(StringLiteral body,
       {String? initialName, bool startAtZero = false}) {
     var messageText = body.toSource();
+    //Strings from constant fields will not pass below condition so initialName will be unchanged.
+    // Needing condition initialName.endsWith('_intlFunction') bcz when any node has addTestId we need to give that
+    // priority, and will not override the node name.
+    if (body is StringInterpolation && (initialName == null || initialName.endsWith('_intlFunction'))) {
+
+      var strings = body.elements.where((each) => each is InterpolationString).map((each) => (each as InterpolationString).value).toList();
+      var data=strings.join(' ').trim();
+      if(data.isNotEmpty && !data.contains("'")){
```

Comment:
Do we need the check for apostrophe? It's not really part of the ticket request, but it seems to me that in the test examples we get better results if we do the same thing with or without an apostrophe. e.g. bobsLastName rather than intlFunction0. And then I think this function might be able to be simplified.

Reply:
Yea, I was just preventing this for the same reason. I can update this to get the same value with or without the apostrophe.

Reply:
This has been updated in latest commit.